tensorflow_datasets/core/download/downloader.py

```diff
@@ -148,21 +149,39 @@ def _sync_download(self, url, destination_path):
         'https': os.environ.get('TFDS_HTTPS_PROXY', None),
         'ftp': os.environ.get('TFDS_FTP_PROXY', None)
     }
+    CA_BUNDLE = os.environ.get(
+        'TFDS_CA_BUNDLE',
+        None) or os.environ.get(
+        'REQUESTS_CA_BUNDLE',
+        None)
+    if CA_BUNDLE:
+      CA_BUNDLE = extract_zipped_paths(CA_BUNDLE)
+    if not hasattr(ssl, '_create_unverified_context'):
+      # For python version <= 2.7.8
+      ssl.__dict__['_create_unverified_context'] = lambda: None
+      ssl.__dict__['create_default_context'] = lambda *args, **kwargs: None
+    CA_VERIFY = {
+        'urllib': ssl._create_unverified_context() if CA_BUNDLE is None or len(CA_BUNDLE) == 0
+        else ssl.create_default_context(capath=CA_BUNDLE),
+        'requests': False if CA_BUNDLE is None else CA_BUNDLE
```

Comment:
what is the type of 'requests' ?

I think you wanted it to be bool, but here it is either False, or path to the bundle, right ? Consider changing to:

'requests': CA_BUNDLE is None

Reply:
Not really. It gives the file path if CA_BUNDLE is not None else it returns False

Reply:
> I'd like variables to have concrete types assigned (rather than "either bool or string" :-)

Sure :)

Reply:
Actually for this parameter, if `verify == False`, it will not verify, if `verify is None`, it will verify using the default system certificates. And if verify is set to any CA Path, it will verify using that property. So, I think even if we go for only one datatype here, I need to perform another if-else check in the and pass the parameters, causing code duplication. :(
What do you suggest? What should I do?